docs/miscellaneous/security.md

```diff
@@ -40,7 +40,7 @@ It is recommended to use an IAM user with the `AdministratorAccess` policy to cr
 
 ### Operator
 
-A process called the Cortex operator runs on your cluster and is responsible for deploying and managing your APIs on the cluster. The operator will use `CLUSTER_AWS_ACCESS_KEY_ID` and `CLUSTER_AWS_SECRET_ACCESS_KEY` as its credentials if specified, otherwise it will default to using `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+A process called the Cortex operator runs on your cluster and is responsible for deploying and managing your APIs on the cluster. The operator will use the designated cluster credentials (e.g. `--cluster-aws-key` or `CLUSTER_AWS_ACCESS_KEY_ID`) if specified, otherwise it will default to using the credentials used to spin up the cluster (e.g. `--aws-key` or `AWS_ACCESS_KEY_ID`).
```

Comment:
nit: consider adding the `$` before the env vars, do you think that makes it more clear?